packages/config/src/index.js

```diff
@@ -8,7 +10,7 @@ const { normalizeConfig } = require('./normalize')
 const { handleFiles } = require('./files')
 const { EVENTS, LEGACY_EVENTS } = require('./events')
 
-const resolveConfig = async function(configFile, { cwd, ...options } = {}) {
+const resolveConfig = async function(configFile, { cwd = getCwd(), ...options } = {}) {
```

Comment:
Nitpick: shouldn't need to change this file since `getConfigPath` already has this behavior.